tenseal/tensors/ckksvector.cpp

```diff
@@ -85,7 +86,13 @@ CKKSVector& CKKSVector::add_inplace(CKKSVector to_add) {
     }
 
     if (this->size() != to_add.size()) {
-        throw invalid_argument("can't add vectors of different sizes");
+        if (this->size() == 1) {
```

Comment:
this check is identical in all methods, maybe it can be separated in a private method.

Reply:
yeah, that's what I was referring to in my comments :point_up: , we should later refactor all the checks into a function with a common error message like "can't operate on vectors of different sizes");